.azure-devops/build-pipeline.yaml

```diff
@@ -25,7 +25,7 @@ resources:
 jobs:
 - job: Windows
   pool:
-    name: OPX Windows 1903 Dogfood Pool
+    name: OPX Windows 1903 Dogfood Pool - GitHub
```

Comment:
This will need to be checked in eventually. You can run the pipeline against a branch manually instead of creating a PR that you will abandon.